content/chainguard/chainguard-images/working-with-images/security-advisories/managing-advisories/index.md

```diff
@@ -0,0 +1,165 @@
+date: 2024-08-05T20:23:51+00:00
+lastmod: 2024-08-05T20:23:51+00:00
+draft: false
+tags: ["Overview", "Product", "Chainguard Images", "CVE"]
+images: []
+menu:
+  docs:
+    parent: "security-advisories"
+weight: 030
+toc: true
+---
+
+Chainguard operates its own [Security Advisories](https://images.chainguard.dev/security/) page to alert users about the status of vulnerabilities found in Chainguard Images. To maintain this database, we use [`wolfictl`](https://github.com/wolfi-dev/wolfictl/), a tool which is used to work with the [Wolfi un-distro](https://github.com/wolfi-dev/).
+
+In this guide, we will walk through using `wolfictl` to create an advisory for a vulnerable package. We’ll also learn how to update this advisory as more information about the vulnerability is disclosed over time. To follow along, you will need to have [‘git’](https://git-scm.com/) and the [go programming language](https://go.dev/dl/) installed on your machine. 
+
+This guide will focus on the packages and advisories issued for Wolfi.
+
+
+## How to Install `wolfictl`
+
+To work with security advisories, we will need to install the `wolfictl` tool to our machine. First, execute the following command in your terminal to clone the `wolfictl` repository locally, and navigate to it. 
+
+```shell
+git clone git@github.com:wolfi-dev/wolfictl.git wolfictl && cd $_
+```
+
+Then, using `go`, install the `wolfictl` command:
+
+```shell
+go install
+```
+If you encounter any errors during installation, your installed version of `go` may be out of date. You can check what version of go you have installed by running `go version` in your terminal. Check the [`go.mod` file](https://github.com/wolfi-dev/wolfictl/blob/main/go.mod) in the `wolfictl` repository to determine what version of `go` you will need, and be sure to update `go` to this version or a later release to continue.
+
+You can verify that you have successfully installed `wolfictl` by executing the `wolfictl version` command in your terminal, with output expected to be shown as follows. Note that the exact output will vary over time as new `wolfictl` versions are released. 
+You may need to add the `wolfictl` binary to your `$PATH` after installation if your system does not recognize the command.
+
+```shell
+wolfictl version
+ __        __   ___    _       _____   ___    ____   _____   _
+ \ \      / /  / _ \  | |     |  ___| |_ _|  / ___| |_   _| | |
+  \ \ /\ / /  | | | | | |     | |_     | |  | |       | |   | |
+   \ V  V /   | |_| | | |___  |  _|    | |  | |___    | |   | |___
+    \_/\_/     \___/  |_____| |_|     |___|  \____|   |_|   |_____|
+wolfictl: A CLI helper for developing Wolfi
+
+GitVersion:    devel
+GitCommit:     6c98dc69a559192575d085d87fd916d8281dd67d
+GitTreeState:  clean
+BuildDate:     2024-07-23T01:57:17
+GoVersion:     go1.22.5
+Compiler:      gc
+Platform:      darwin/arm64
+```
+
+
+## Cloning Package and Advisory Repositories
+
```

Comment:
This section almost looks unfinished. It's begging for some transitions and context between the code blocks and before the next section.